src/Components/CheckTypeDropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import {useEffect, useState} from 'react';
+import {ChevronDownIcon} from '@heroicons/react/20/solid';
+import {SimpleButton} from './Tailwind/Button';
+
+export interface CheckType {
+  id: number;
+  title: string;
+}
+
+interface EventCheckTypeDropdownProps {
+  values: CheckType[];
+  selected: CheckType;
+  onChange: (v: CheckType) => void;
+}
+
+function EventCheckTypeDropdown({values, selected, onChange}: EventCheckTypeDropdownProps) {
```

Comment:
```suggestion
function EventCheckInTypeDropdown({values, selected, onChange}: EventCheckTypeDropdownProps) {
```

This is for consistency.